dev/com.ibm.ws.transport.http/resources/com/ibm/ws/http/channel/internal/resources/httpchannelmessages.nlsprops

```diff
@@ -177,3 +177,11 @@ cookies.samesite.unsupportedWildcard.useraction=Ensure that only a single wildca
 cookies.samesite.knownDuplicateName=CWWKT0037W: A cookie name or pattern [{0}], which is marked as a duplicate, was found in the SameSite [{1}] configuration. The [{0}] cookie name or pattern is ignored. Any cookie name or pattern that is defined by the lax, none, and strict configurations can be defined in only one of the three configurations.
 cookies.samesite.knownDuplicateName.explanation=The specified value was already found by the configuration and mapped to a SameSite value.
 cookies.samesite.knownDuplicateName.useraction=Ensure that cookie names and patterns are unique across the lax, none, and strict configuration lists.
+
+trusted.header.origin.invalid.value=CWWKT0038W: The configured trustedHeaderOrigin value [{0}] is invalid and will not be used.
```

Comment:
**change**
The configured trustedHeaderOrigin value [{0}] is invalid and will not be used.
**to**
The [{0}] configured trustedHeaderOrigin value is invalid and cannot be used.